src/mscorlib/src/System/Collections/Comparer.cs

```diff
@@ -41,18 +41,6 @@ public sealed class Comparer : IComparer , ISerializable
             Contract.EndContractBlock();
             m_compareInfo = culture.CompareInfo;
         }
-        
-        private Comparer(SerializationInfo info, StreamingContext context) {            
```

Comment:
Removing this will break deserialization.  Do CoreFx tests pass with this change?  If so, that's a gap in our tests.